code/scripts/index.js

```diff
@@ -56,20 +56,29 @@ document.addEventListener('DOMContentLoaded', function() {
               stripePaymentToken: response.id
             })
 
+            let url = config.createSubscriptionUrl
+            let successFunction = (data) => {
+              if (window.mixpanel) mixpanel.track('Started subscription', { topic: name })
```

Comment:
Could we add a corresponding 'Updated payment card' mixpanel event if they've updated their card details please?